tests/Integration/ClientTest.php

```diff
@@ -230,15 +231,46 @@ public function testSearch(): void
             $this->client->indexDocument(self::TEST_INDEX, 'document-id', ['uuid' => 'this-is-a-uuid', 'payload' => []], ['refresh' => 'true'])
         );
         $query = [
-            'term' => [
-                'uuid.keyword' => [
-                    'value' => 'this-is-a-uuid'
+            'query' => [
+                'term' => [
+                    'uuid.keyword' => [
+                        'value' => 'this-is-a-uuid'
+                    ]
+                ]
+            ]
+        ];
+        $response = Promise\wait($this->client->search($query));
```

Comment:
This is a BC break. Now the argument passed to `$this->client->search()` must have as root element `query`. We should create an `UPGRADE-1.x-2.0.md` document to track these changes.